test/RewardsStreamerMP.t.sol

```diff
@@ -760,7 +839,7 @@ contract StakeTest is RewardsStreamerMPTest {
         );
 
         uint256 currentTime = vm.getBlockTimestamp();
-        uint256 timeToMaxMP = _calculateTimeToMPLimit(stakeAmount);
+        uint256 timeToMaxMP = _calculateTimeToMP(stakeAmount, totalMaxMP - totalMP);
```

Comment:
Any reason this is changed? I think reading "calculate time to max MP" is more intuitive than what this change is doing.

Reply:
Because calculateTimeToMP is more generic, and can be reused for other tests. While TimeToMPLimit would hardcode this function to always calculate time to max mp.

Reply:
We should at least rename this. "time to MP" doesn't make sense. I no longer understand what the function is doing without looking at its source.